src/main/java/com/blackducksoftware/integration/hub/alert/channel/hipchat/HipChatChannel.java

```diff
@@ -99,17 +99,16 @@ public String testGlobalConfig(final GlobalHipChatConfigEntity entity) {
 
                 // The {"message":"test"} is required to avoid a BAD_REQUEST (OkHttp issue: #854)
```

Comment:
If hub-common no longer uses OkHttp, we may be able to get rid of the hard-coded string in the request. Not a high priority, but worth noting.